css/styles.css

```diff
@@ -2,678 +2,526 @@
-    color: white;
-  }
+.home-page p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
+}
 
-  .pointer a:hover {
-    cursor: pointer;
-    color: rgb(190, 190, 214);
-  }
+.home-page button {
+  background: none;
+}
 
-  .pointer {
-    margin-top: 80px;
-    color: white;
-  }
+.desktop-navigation {
+  display: none;
+}
 
-  .page-1 {
-    display: grid;
-    grid-template-columns: 100%;
-    grid-template-rows: 0.1fr 0.1fr 0.1fr 1fr 1fr 1fr 1fr 1fr 1fr;
-    grid-template-areas:
-      "header"
-      "line"
-      "grid"
-      "grid"
-      "grid"
-      "grid"
-      "grid"
-      "grid";
-  }
+.container {
+  width: 90%;
+  max-width: 900px;
+  margin: 0 auto;
+}
 
-  .abt-card {
-    display: grid;
-    grid-template-columns: 100%;
-    grid-template-rows: 1fr 1fr 1fr;
-    grid-template-areas:
-      "card"
-      "card"
-      "card";
-    gap: 1px;
-  }
+.container-model {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
 
-  .white-color {
-    color: rgb(129, 129, 129);
-    size: 200%;
-  }
+.social-media {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  width: 170px;
+}
 
-  .card h1 {
-    color: #344563;
-    font-family: "Inter", sans-serif;
-    font-style: normal;
-    font-weight: 600;
-    font-size: 24px;
-    line-height: 32px;
-    text-align: center;
-    align-items: center;
-  }
+.social-media > ul > li > a {
+  text-decoration: none;
+  color: #42526e;
+  font-size: 70px;
+  margin-top: 1.5em;
 }
 
-.header h1 {
-  font-family: "Inter", sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  color: #172b4d;
-  margin-top: 50px;
+/* Card works group */
+.recent-works {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 200px repeat(5, 6fr);
+  text-align: center;
+  width: 90%;
+  margin: 0 auto;
 }
 
-hr.line {
-  display: none;
-  background: #36b37e;
-  border-radius: 24px;
-  margin-left: 45%;
-  width: 48px;
+.recent-works h1:first-child {
+  padding: 1em;
+}
+
+.recent-works h1:first-child::after {
+  content: "";
   height: 4px;
-  order: 1;
-  flex: none;
-  flex-grow: 0;
-  border-top: 1px solid red;
+  width: 48px;
+  margin: 0 auto;
+  display: block;
+  position: relative;
+  top: 20px;
+  z-index: -1;
+  background: #36b37e;
+  border-radius: 8px;
 }
 
-.grid {
+.card-work {
+  overflow: hidden;
   display: flex;
   flex-direction: column;
-  align-items: flex-end;
-  margin: 30px;
+  justify-content: center;
+  text-align: center;
+  padding: 0;
+  margin: 1em auto;
+  width: 327px;
+  height: 474px;
   border: 1px solid #d0d9d4;
   border-radius: 8px;
-  background: #ebf0ee;
 }
 
-.grid1 {
-  background-image: url(../images/background-contact-shape2.png);
-  height: 300px;
-  border-radius: 5px;
+.card-work > .card-tags > li {
+  background: rgb(226, 225, 225);
```

Comment:
Kindly fix the color of the background for the cards-tags here like on the Figma template.
<img width="845" alt="Screenshot 2023-02-26 at 12 04 38 AM" src="https://user-images.githubusercontent.com/25887253/221380197-87999dba-f294-4d70-8a62-a2f762007a46.png">

Reply:
Done